traffic_portal/app/src/common/api/CDNService.js

```diff
@@ -240,6 +240,23 @@ var CDNService = function($http, $q, Restangular, locationUtils, messageModel, E
         return request.promise;
     };
 
+	this.regenerateKSK = function(kskRequest, cdnKey) {
+		var request = $q.defer();
+
+		$http.post(ENV.api['root'] + cdnKey + "/dnsseckeys/ksk/generate", kskRequest)
```

Comment:
This needs to hit `/api/1.4/cdns/my-cdn-name/dnsseckeys/ksk/generate`. It's currently hitting `/api/1.3/my-cdn-name/dnsseckeys/ksk/generate`. So, `/cdns/` needs added to the path. 

I'm not sure where `ENV.api['root']` is coming from, it might be a config file that needs updated to `1.4`. Wherever it is, it needs to default to `1.4` now, since 1.4 is required for this endpoint.